scripts/build-data.js

```diff
@@ -6,16 +6,88 @@ const path = require("path");
 const flatten = require("lodash/flatten");
 const flattenDeep = require("lodash/flattenDeep");
 const mapValues = require("lodash/mapValues");
+const pickBy = require("lodash/pickBy");
 const pluginFeatures = require("../data/plugin-features");
 const builtInFeatures = require("../data/built-in-features");
 
 const renameTests = (tests, getName) =>
   tests.map((test) => Object.assign({}, test, { name: getName(test.name) }));
 
-const es6Data = require("compat-table/data-es6");
-const es6PlusData = require("compat-table/data-es2016plus");
+// The following is adapted from compat-table:
+// https://github.com/kangax/compat-table/blob/gh-pages/build.js
+//
+// It parses and interpolates data so environments that "equal" other
+// environments (node4 and chrome45), as well as familial relationships (edge
+// and ie11) can be handled properly.
+
 const envs = require("compat-table/environments");
 
+const byTestSuite = (suite) => (browser) => {
+  return Array.isArray(browser.test_suites)
+    ? browser.test_suites.indexOf(suite) > -1
+    : true;
+};
+
+const es6 = require("compat-table/data-es6");
+es6.browsers = pickBy(envs, byTestSuite("es6"));
+
+const es2016plus = require("compat-table/data-es2016plus");
+es2016plus.browsers = pickBy(envs, byTestSuite("es2016plus"));
+
+const interpolateAllResults = (rawBrowsers, tests) => {
+  const interpolateResults = (res) => {
+    let browser;
+    let prevBrowser;
+    let result;
+    let prevResult;
+    let bid;
+    let prevBid;
+
+    for (bid in rawBrowsers) {
```

Comment:
This could be `for (const bid in rawBrowsers)` and then the above `let bid` declaration is not needed.